### PR TITLE
bump cloud resources operator to 0.5.0

### PR DESCRIPTION
**Pull request by austincunningham**
bump the cloud resources operator

### CODE REVIEW

**Comment by dimitraz** [comment]
:eyes:

File: manifests/integreatly-cloud-resources/cloud-resources-0.5.0/cloud-resources.v0.5.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -8,67 +8,79 @@ metadata:
           "apiVersion": "integreatly.org/v1alpha1",
           "kind": "Redis",
           "metadata": {
+            "labels": {
+              "productName": "productName"
```
Comment (dimitraz) [inline manifests/integreatly-cloud-resources/cloud-resources-0.5.0/cloud-resources.v0.5.0.clusterserviceversion.yaml]: Quick question, will there be another PR for labelling our resources with the product name?

**Comment by ciaranRoche** [comment]
Closing as work duplicated - https://github.com/integr8ly/integreatly-operator/pull/308
